src/lib/swapper/swappers/ThorchainSwapper/endpoints.ts

```diff
@@ -28,8 +29,9 @@ export const thorchainApi: Swapper2Api = {
   ): Promise<Result<TradeQuote2[], SwapErrorRight>> => {
     const { receiveAddress } = input
 
-    // TEMP: thorchain swapper is set at 25 bps
-    const affiliateBps = '25'
+    const thorSwapAffiliateFees = getConfig().REACT_APP_FEATURE_THOR_SWAP_AFFILIATE_FEES
```

Comment:
Maybe a boolean-like name,

```suggestion
    const applyThorSwapAffiliateFees = getConfig().REACT_APP_FEATURE_THOR_SWAP_AFFILIATE_FEES
```